body.tex

```diff
@@ -47,12 +47,15 @@ \subsection{Data Preview 0.2}\label{sec:dp02}
 
 
 \section{Community engagement }
-The Rubin Community Engagement Team (CET) provided support to delegates.
-Biweekly virtual “Delegate Assemblies” with hands-on demonstrations were held, and continue monthly.
-DP0 documentation was mentioned above there were also many tutorial Notebooks.
-A dedicated \href{https://community.lsst.org/c/support/dp0/}{Community.lsst.org category: “Support -- Data Preview 0”} was created.
-This is also publicly accessible to non-delegates.
+The Rubin Community Engagement Team (CET) provided documentation and support to the scientists and students participating in DP0 as delegates.
+The term 'DP0 delegates' was adopted to reflect DP0 participants’ important roles of representing the broad science community as learners, testers, and providers of feedback, and of sharing the benefits of their DP0 participation with their communities as teachers and colleagues.
 
+Documentation at dp0-1.lsst.io included a delegate's homepage listing all the resources, events, guidelines, and expectations; descriptions of the data products and table schema; instructions for accessing the RSP's three aspects and usage risks; and tutorials to demonstrate simple science investigations with the DP0 data set at the RSP. 
+Delegate resources included biweekly virtual “Delegate Assemblies” with hands-on demonstrations and breakout rooms to facilitate co-working; a dedicated category in the Rubin Community Forum (Community.lsst.org) where delegates can ask questions and discuss their DP0 work; and a GitHub repository where delegates can contribute to shared code and notebooks.
+GitHub Issues were also used as a means for delegates to request technical assistance.
+
+Rubin staff from the CET and Data Production worked together to respond to delegate's requests for support and bug reports.
```

Comment:
For lessons learned or thoughts - will this scale up to 10K users from 200 hundred ? Do we need to do anything differently.